src/test/java/alihan/string1_test/MakeTagsTest.java

```diff
@@ -0,0 +1,40 @@
+package alihan.string1_test;
+
+import com.digital_nomads.alihan.string1.MakeTags;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class MakeTagsTest {
+
+    @Test(dataProvider = "makeTags")
+    public void testMakeTags (String tag, String word, String expected){
+        MakeTags makeTags = new MakeTags();
+        String result = makeTags.makeTags(tag,word);
+        Assert.assertEquals(result,expected);
+
+    }
+    @DataProvider
+    public Object[][] makeTags(){
+        return new Object[][]{
+
```

Comment:
tut body poprav

Reply:
@Asylbash here is just a page break on Github )))

Reply:
nado je vse poprobyvat